inference-engine/src/mkldnn_plugin/ngraph_transformations/rnn_sequences_optimization.cpp

```diff
@@ -113,22 +122,39 @@ MKLDNNPlugin::OptimizeRNNSequenceTransposes::OptimizeRNNSequenceTransposes() {
 
 MKLDNNPlugin::OptimizeLSTMSequenceTransposes::OptimizeLSTMSequenceTransposes() {
     ngraph::matcher_pass_callback callback = [](ngraph::pattern::Matcher &m) {
-        auto lstmSequence = std::dynamic_pointer_cast<ngraph::op::v5::LSTMSequence>(m.get_match_root());
+        std::shared_ptr<ngraph::Node> lstmSequence;
+        // Bidirectional cases are not supported
+        auto lstmSequence5 = std::dynamic_pointer_cast<ngraph::op::v5::LSTMSequence>(m.get_match_root());
+        if (lstmSequence5) {
+            if (lstmSequence5->get_direction() == ngraph::op::RecurrentSequenceDirection::BIDIRECTIONAL) {
+                return false;
+            }
+            lstmSequence = lstmSequence5;
+        }
+
+        auto lstmSequence1 = std::dynamic_pointer_cast<ngraph::op::v0::LSTMSequence>(m.get_match_root());
```

Comment:
please use opset everywhere, not op::v0::
for this line:
 ngraph::opset1::LSTMSequence

Reply:
Fixed, thank you